defaults/main.yml

```diff
@@ -14,17 +14,15 @@ logging_inputs: []
 # List of flows defining the input and the output relationships
 logging_flows: []
 
-# If logging_purge_confs is set to true, existing config files will be purged.
+# `logging_purge_confs` is set to `true`, remove files in rsyslog.d which do
+# not belong to any rpm packages. That includes config files generated by the
+# previous logging role run.
 logging_purge_confs: false
 
-# Variable to specify the directory to store backup file which contains pre-
-# logging system role run. The backup is used when logging_restore_confs is
-# set to true.
-# Default to '/var/lib/rsyslog' for rsyslog.
-logging_backup_dir: ""
-
-# If logging_restore_confs is set to true, backed up config files are restored.
-logging_restore_confs: false
+# If `logging_reset_confs` is set to `true`, reinstall rsyslog package, and
+# restore the pre-existing config files, which restores the default
+# `/etc/rsyslog.conf`.
+logging_reset_confs: false
```

Comment:
This represents a breaking API change - we are removing an existing variable.  Instead, I think we should reuse the variable `logging_restore_confs` and just change the way it makes the role behave.

Reply:
`logging_restore_confs` was added in this commit (tag: 1.7.0)
```
commit 522da0a0a504d50f7f4397ac36a038bdac390c0d (tag: 1.7.0, origin/master, origin/HEAD, master)
Author: Noriko Hosoi <nhosoi@redhat.com>
Date:   Thu Jan 6 15:33:10 2022 -0800
    Add logging_restore_confs variable to restore backup.
```
The tagged logging source has not been used in any linux-system-roles build yet.
This is the snippet of the rawhide spec file.
```
#%%defcommit 10 20dd3e5520ca06dcccaa9b3f1fb428d055e0c23f
%global rolename10 logging
%deftag 10 1.6.1
```
So, I think it's safe to rename the variable name. (And `logging_reset_confs` represents the behavior better... :)